src/runtime/cpp/lean_runtime.cpp

```diff
@@ -202,11 +216,17 @@ void obj_cell::dealloc() {
             case obj_kind::Closure:
                 DEC_FIELDS(it, todo);
                 it->~obj_cell();
-                g_obj_pool->recycle(it, sz+1);
+                // trying to recycle closures and raw ptrs causes crashes
```

Comment:
Trying to recycle either a Closure or my new RawPtr type causes a bad access to be trigger when attempting to clean up the free list.